interfaces/apparmor/backend.go

```diff
@@ -63,6 +70,138 @@ func (b *Backend) Name() interfaces.SecuritySystem {
 	return interfaces.SecurityAppArmor
 }
 
+// Initialize prepares customized apparmor policy for snap-confine.
+func (b *Backend) Initialize() error {
+	// NOTE: It would be nice if we could also generate the profile for
+	// snap-confine executing from the core snap, right here, and not have to
+	// do this in the Setup function below. I sadly don't think this is
+	// possible because snapd must be able to install a new core and only at
+	// that moment generate it.
+	return setupSnapConfineGeneratedPolicy()
+}
+
+// setupSnapConfineGeneratedPolicyImpl inspects the system and sets up local
+// apparmor policy for snap-confine. Local policy is included by the
+// system-wide policy. If the local policy has changed then the apparmor
+// profile for snap-confine is reloaded.
+func setupSnapConfineGeneratedPolicyImpl() error {
+	// Create the local policy directory if it is not there.
+	if err := os.MkdirAll(dirs.SnapConfineAppArmorDir, 0755); err != nil {
+		return fmt.Errorf("cannot create snap-confine policy directory, %s", err)
```

Comment:
s/,/:/. Every error and log message in this PR seems to use this convention, and this is not our usual convention. Can you please review all error messages and make sure they follow suit?

As a more general note, let's please be careful to preserve consistency in the code base. I've seen that sort of minor discrepancy often, and if it goes unnoticed the code gradually becomes messy. We can change conventions, but we cannot change them in isolation and without notice.

Reply:
I was wondering why I got this wrong. I must have got the impression that the `cannot quux: %s` is the wrong way and `cannot quux, %s` is the right way of formatting errors somehow (as I tried to be consistent, just going the wrong way).

I've reviewed and corrected all the error messages around this code.